tests/NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests/Steps/OrderSteps.cs

```diff
@@ -60,26 +61,37 @@ public async Task ThenTheOrdersListIsReturnedWithTheFollowingValues(Table table)
 
             var orders = (await _response.ReadBodyAsJsonAsync()).Select(CreateOrders);
 
-            orders.Should().BeEquivalentTo(expectedOrders, a => a.Excluding(u => u.OrderStatusId));
+            orders.Should().BeEquivalentTo(expectedOrders,
+                orderTable => orderTable.Excluding(order => order.OrderStatusId)
+                    .Excluding(order => order.OrganisationId));
+        }
+
+        [Then(@"an empty list is returned")]
+        public async Task AnEmptyListIsReturned()
+        {
+            var orders = (await _response.ReadBodyAsJsonAsync()).Select(CreateOrders);
+            orders.Count().Should().Be(0);
         }
 
         private static object CreateOrders(JToken token)
         {
             return new
             {
                 OrderId = token.SelectToken("orderId").ToString(),
-                Description = token.SelectToken("orderDescription").ToString(),
+                Description = token.SelectToken("description").ToString(),
                 Status = token.SelectToken("status").ToString(),
                 LastUpdated = token.SelectToken("lastUpdated").ToObject<DateTime>(),
                 LastUpdatedBy = token.SelectToken("lastUpdatedBy").ToObject<Guid>(),
                 Created = token.SelectToken("dateCreated").ToObject<DateTime>()
```

Comment:
I think we should be using the Value<T> method for extracting values from JTokens as they don't throw if the desired token is null.

Like so: 
```
                OrderId = token.SelectToken("orderId").ToString(),
                Description = token.SelectToken("description").ToString(),
                Status = token.SelectToken("status").ToString(),
                LastUpdated = token.SelectToken("lastUpdated").ToObject<DateTime>(),
                LastUpdatedBy = token.SelectToken("lastUpdatedBy").ToObject<Guid>(),
                Created = token.SelectToken("dateCreated").ToObject<DateTime>()
```
Becomes:
```
                OrderId = token.Value<string>("orderId"),
                Description = token.Value<string>("description"),
                Status = token.Value<string>("status"),
                LastUpdated = token.Value<DateTime>("lastUpdated"),
                LastUpdatedBy = token.Value<Guid>("lastUpdatedBy"),
                Created = token.Value<DateTime>("dateCreated")
```

Reply:
👍

Reply:
Although, note: I've had to do it like this, as the conversion to GUID did not work through token.Value<Guid>

Reply:
return new
            {
                OrderId = token.Value<string>("orderId"),
                Description = token.Value<string>("description"),
                Status = token.Value<string>("status"),
                LastUpdated = token.Value<DateTime>("lastUpdated"),
                LastUpdatedBy = token.SelectToken("lastUpdatedBy").ToObject<Guid>(),
                Created = token.Value<DateTime>("dateCreated")
            };